README.rst

```diff
@@ -48,6 +48,10 @@ installed using ::
 To install the remaining packages see the installation instructions
 at their respective web pages.
 
+The Python3 PDFgui project filename extension has been changed to .ddp3.
+Please note that sometimes it may fail for .ddp project files that were
```

Comment:
`Please note that the Python3 PDFgui will read .ddp3 files.  It is also possible for it to read .ddp files that were saved by the Python2 PDFgui but it will sometimes fail to read these.  We are working on a solution that will be available in a future version.`